### ISSUE TITLE
Install request: EIGENSOFT

### ISSUE THREAD

**Issue by heatherkellyucl** [UCL-RITS/rcps-buildscripts#39]
IN01193402

https://data.broadinstitute.org/alkesgroup/EIGENSOFT/

Info:
http://www.hsph.harvard.edu/alkes-price/software/
http://www.hsph.harvard.edu/alkes-price/eigensoft-frequently-asked-questions/

Does not appear to use the Broad license so shouldn't need a separate application group:

> SOFTWARE COPYRIGHT NOTICE AGREEMENT
This software and its documentation are copyright (2010) by Harvard University 
and The Broad Institute. All rights are reserved. This software is supplied 
without any warranty or guaranteed support whatsoever. Neither Harvard 
University nor The Broad Institute can be responsible for its use, misuse, or 
functionality. The software may be freely copied for non-commercial purposes, 
provided this copyright notice is retained.

Registration is voluntary and for update information not license purposes. (From http://www.hsph.harvard.edu/alkes-price/software/)

> SOFTWARE REGISTRATION: we encourage EIGENSOFT users to register here. Registration is voluntary, but will allow us to send information about software updates.

**Comment by heatherkellyucl** [comment]
The Makefile as provided doesn't work with the Intel compilers:

```
/shared/ucl/apps/openblas/0.2.14/intel-2015-update2/lib/libopenblas.so: undefined reference to `for_cpystr'
/shared/ucl/apps/openblas/0.2.14/intel-2015-update2/lib/libopenblas.so: undefined reference to `for_concat'
```

Doesn't link with ifort or link `-lirc`. Adding that manually to LDLIBS gives dgsrv errors. Built with GCC instead - requires to add `-lm -lpthread` to LDLIBS, then it will compile and the pcatoy tiny example runs.

```
module load gcc-libs
module load compilers/gnu/4.9.2
module load gsl/1.16/gnu-4.9.2
module load openblas/0.2.14/gnu-4.9.2

cd src
make clobber
make install OPENBLAS=$OPENBLASROOT LDLIBS="-lgsl -lopenblas -lgfortran -lrt -lm -lpthread"
```

No install prefix option so have to build in place.

**Comment by heatherkellyucl** [comment]
Eigensoft is available on Legion and Grace as `eigensoft/6.1.1/gnu-4.9.2`

**Comment by heatherkellyucl** [comment]
Wiki page created at https://wiki.rc.ucl.ac.uk/wiki/EIGENSOFT_on_RC_Systems
